libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateWritableButton.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+** EPITECH PROJECT, 2022
+** rtype
+** File description:
+** CreateWritableButton
+*/
+
+#include "CreateWritableButton.hpp"
+#include <mutex>
+#include "R-TypeLogic/Global/Components/ButtonComponent.hpp"
+#include "R-TypeLogic/Global/Components/LayerLvL.hpp"
+#include "R-TypeLogic/Global/Components/PositionComponent.hpp"
+#include "R-TypeLogic/Global/Components/SizeComponent.hpp"
+#include "GraphicECS/SFML/Components/AssociatedIdComponent.hpp"
+#include "GraphicECS/SFML/Components/WritableButtonActionComponent.hpp"
+
+namespace ecs
```

Comment:
Is it a good namespace "ecs" ?
is it "R-TypeLogic::EntityManipulation::CreateEntitiesFunctions" ?

Reply:
Refer to discord conversation